### PR TITLE
Remove project fields

### PR DESCRIPTION
**Pull request by buchanae**
The `project` field is underspecified – is it required to be a cloud project ID? Or just more metadata? Or both? It could be defined via the `tags` field, which would make it simpler to filter via `ListTasks` and might fit to #84 which uses `tags` for cloud/deployment  specific resource requests.

### CODE REVIEW

**Comment by geoffjentry** [comment]
FWIW for a system we're putting on top of Cromwell we went with the tag route. We found that `project` was both too limiting as well as too generically named, as you note yourself.

We decided that systems sitting on top of Cromwell could set tags to their hearts content and work that way and that way clients could have whatever they want.

**Comment by geoffjentry** [comment]
tl;dr that's a :+1: for tags instead of project

**Comment by pditommaso** [comment]
👍
